CHANGELOG-1.2.md

```diff
@@ -0,0 +1,10 @@
+# Changelog since v1.1.0
+
+## Deprecations
+
+## Notable Features
+
+* Support attachment of inline volumes migrated to CSI [#153](https://github.com/kubernetes-csi/external-attacher/pull/154)
+* Action required: PATCH is used instead of POST to modify VolumeAttachments on API servers. Please note changes in the external attacher RBAC policy. [#149](https://github.com/kubernetes-csi/external-attacher/pull/149)
```

Comment:
I believe this is considered a breaking change as part of our still yet to be finalized upgrade policy, ie you can't upgrade just the container image. In fact, our canary tests are failing maybe because of this? https://k8s-testgrid.appspot.com/sig-storage-csi#canary-on-1-14

How much do we really want this change? If we want to keep it, we may need to bump the major version.

Reply:
yeah it was causing GCE PD tests to fail as well. Better to rip off the bandaid quickly rather than leave it for later?
I already updated the RBAC for PD to include PATCH as well :)

Reply:
If we want to keep the change, then we should call this release 2.0 to indicate that you may need to take some other actions to make it work.